### DISCUSSION TITLE
Two similar Hamlet quotes

### DISCUSSION THREAD

**Discussion by LukeNK** [monkeytypegame/monkeytype#4126] [Questions]
In English quotes, there are two Hamlet quotes: 3676 and 6518, both are the same:

> To be, or not to be? That is the question: Whether 'tis nobler in the mind to suffer The slings and arrows of outrageous fortune, or to take arms against a sea of troubles, and, by opposing, end them?

The reason I submit a question is that I don't know if should I remove one of them and shift the quotes after it by one; or simply remove it; or even left it as it is. I know it is a minor issue but I think it is better if we remove one of the two quotes, and if we remove one, which one should we remove?

**Comment by unrealapex** [comment]
EDIT: see this [comment](https://github.com/monkeytypegame/monkeytype/pull/4128#issuecomment-1493377999)

<s>Each `id` value should be different so the best approach would be to remove the duplicate and shift the ids.</s>

https://github.com/monkeytypegame/monkeytype/pull/4136
